tools/cmd/prepare_prebuilt_workers/main.go

```diff
@@ -137,26 +137,32 @@ func main() {
 
 			// build image
 			log.Println(fmt.Sprintf("building %s image", lang))
-			buildDockerImage := exec.Command("docker", "build", dockerfileLocation, "-t", image, "--build-arg", fmt.Sprintf("GITREF=%s", gitRef), "--build-arg", fmt.Sprintf("BREAK_CACHE=%s", uniqueCacheBreaker))
+			// TODO(jtattermusch): allow setting REPOSITORY arg to allow building from user forks e.g. https://github.com/USER_NAME/grpc.git
+			buildCommandTimeoutSeconds := 30 * 60 // 30 mins should be enough for all languages
```

Comment:
It should be enough to finish any language on kokoro machines. I am okay with a hard coded timeout, just wondering if it would be useful to make it a flag with default value.

Reply:
I will leave as is in this PR to get the fix in ASAP, but feel free to address as a followup PR.